contrib/labstack/echo.v4/echotrace.go

```diff
@@ -38,15 +38,19 @@ func Middleware(opts ...Option) echo.MiddlewareFunc {
 				tracer.Tag(ext.HTTPURL, request.URL.Path),
 				tracer.Measured(),
 			}
+			finishOptions := []tracer.FinishOption{}
```

Comment:
Can you define this as below?
```suggestion
			var finishOpts []tracer.FinishOption
```
And please move it above (new) line 49, where it is used, without introducing any new empty lines please.